core/src/com/unciv/ui/worldscreen/unit/UnitTable.kt

```diff
@@ -81,6 +70,20 @@ class UnitTable(val worldScreen: WorldScreen) : Table(){
                     worldScreen.mapHolder.setCenterPosition(it, false, false)
                 }
             }
+
+            addSeparator().actor!!
+            add(Table().apply {
```

Comment:
Please separate this to a val, like val closeAndSwitchUnitsTable, and then add() it -this is to reduce nesting and increase readability